src/vma/dev/cq_mgr.cpp

```diff
@@ -526,8 +526,10 @@ mem_buf_desc_t* cq_mgr::process_cq_element_rx(vma_ibv_wc* p_wce)
 	// Get related mem_buf_desc pointer from the wr_id
 	mem_buf_desc_t* p_mem_buf_desc = (mem_buf_desc_t*)(uintptr_t)p_wce->wr_id;
 
-	bool bad_wce = (p_wce->status != IBV_WC_SUCCESS) ||
-			(m_b_is_rx_csum_on && !vma_wc_rx_csum_ok(*p_wce));
+	bool bad_wce = p_wce->status != IBV_WC_SUCCESS;
+	if  (!safe_mce_sys().rx_sw_csum) {
```

Comment:
probably better to use a member to store value of 'safe_mce_sys().rx_sw_csum'.

initializing a `const bool m_is_rx_sw_csum_enabled(safe_mce_sys().rx_sw_csum)` in cq_mgr constructor is save a few small CPU cycles. it is used few time in this code.